Areas/Identity/Data/WhaleSpottingContext.cs

```diff
@@ -0,0 +1,26 @@
+using System;
```

Comment:
I'm guessing these were just created for you, it surprises me that we need these as we have our own context files but if it works locally I can go with it!

Reply:
the build fails without this file so it is used!